internal/updater/checker.go

```diff
@@ -29,28 +34,33 @@ type Checker struct {
 var DefaultChecker = newDefaultChecker()
 
 func newDefaultChecker() *Checker {
+	infoURL := constants.APIUpdateInfoURL
+	if url, ok := os.LookupEnv("_TEST_UPDATE_INFO_URL"); ok {
+		infoURL = url
+	}
```

Comment:
For integration tests we should just use the actual endpoint. For unit tests we shouldn't need env vars, the code structure should facilitate testing. eg. just invoke `NewChecker()` from your tests instead of using `newDefaultChecker()`.

Reply:
Yeah.  We already have a story to remove all of these mocked integration tests: https://www.pivotaltracker.com/story/show/177957411
The idea was to do this story, once we have a multi-file update on the release branch, as we cannot really test that we can do all the installs and updates there, before we have an update deployed on s3.  I think that is still a good idea to keep it around until then...  But you can convince me otherwise.  After all, we have tested our update mechanism on the beta channel quite a bit already.